test/libsyncengine/reconciliation/operation_generator/testoperationgeneratorworker.cpp

```diff
@@ -179,27 +184,23 @@ void TestOperationGeneratorWorker::testCreateOp() {
 void TestOperationGeneratorWorker::testCreateOpWithPseudoConflict() {
     LOGW_DEBUG(_logger, L"$$$$$ TestOperationGeneratorWorker::testCreateOpWithPseudoConflict $$$$$");
     // Simulate file creation on both replica
-    SyncTime createdAt = 1654788079;
-    SyncTime lastmodified = 1654788079;
-    int64_t size = 12345;
-
     std::shared_ptr<Node> lNodeAA = _syncPal->updateTree(ReplicaSide::Local)->getNodeById("lAA");
-    std::shared_ptr<Node> lNodeAAB =
-        std::shared_ptr<Node>(new Node(std::nullopt, _syncPal->updateTree(ReplicaSide::Local)->side(), Str("AAB"), NodeType::File,
-                                       OperationType::Create, "lAAB", createdAt, lastmodified, size, lNodeAA));
+    std::shared_ptr<Node> lNodeAAB = std::make_shared<Node>(
+        std::nullopt, _syncPal->updateTree(ReplicaSide::Local)->side(), Str("AAB"), NodeType::File, OperationType::Create, "lAAB",
+        testhelpers::defaultTime, testhelpers::defaultTime, testhelpers::defaultFileSize, lNodeAA);
     CPPUNIT_ASSERT(lNodeAA->insertChildren(lNodeAAB));
     _syncPal->updateTree(ReplicaSide::Local)->insertNode(lNodeAAB);
 
     std::shared_ptr<Node> rNodeAA = _syncPal->updateTree(ReplicaSide::Remote)->getNodeById("rAA");
-    std::shared_ptr<Node> rNodeAAB =
-        std::shared_ptr<Node>(new Node(std::nullopt, _syncPal->updateTree(ReplicaSide::Remote)->side(), Str("AAB"),
-                                       NodeType::File, OperationType::Create, "rAAB", createdAt, lastmodified, size, rNodeAA));
+    std::shared_ptr<Node> rNodeAAB = std::make_shared<Node>(
+        std::nullopt, _syncPal->updateTree(ReplicaSide::Remote)->side(), Str("AAB"), NodeType::File, OperationType::Create,
+        "rAAB", testhelpers::defaultTime, testhelpers::defaultTime, testhelpers::defaultFileSize, rNodeAA);
     CPPUNIT_ASSERT(rNodeAA->insertChildren(rNodeAAB));
     _syncPal->updateTree(ReplicaSide::Remote)->insertNode(rNodeAAB);
 
     _syncPal->_operationsGeneratorWorker->execute();
 
-    CPPUNIT_ASSERT(_syncPal->_syncOps->size() > 0);
+    CPPUNIT_ASSERT(!_syncPal->_syncOps->isEmpty());
```

Comment:
Do we need a public accessor for `syncOps`?

Reply:
Done